modules/acs-architecture-overview.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * architecture/index.adoc
+
+:_module-type: REFERENCE
+[id="acs-architecture_{context}"]
+= {product-title} architecture
+
+{product-title} installs as a pod in your OpenShift or a Kubernetes cluster and it includes the following components:
```

Comment:
@vikram-redhat How have other teams (like ROSA) been handling references to "OpenShift"? (Since we normally want people use {product-title} instead of just "OpenShift", but these distros use {product-title} for their own use. Are they okay to just use "OpenShift" generically?

Reply:
I would prefer if we use a common-attributes.adoc module to define this value.